.github/workflows/docker.yml

```diff
@@ -76,10 +81,14 @@ jobs:
               PYTHON_VERSION=python3.10
           - sytest_image_tag: buster
             dockerfile: dendrite
-            tags: "matrixdotorg/sytest-dendrite:go113,matrixdotorg/sytest-dendrite:latest"
+            tags: "matrixdotorg/sytest-dendrite:go118,matrixdotorg/sytest-dendrite:latest"
```

Comment:
What's the story here---is this necessary or a drive-by bump?

Reply:
I don't have an explanation of this one, I couldn't find the go113 tag used anywhere so I bumped it to align with the go version.

Reply:
I've reverted the bump to Go 1.18 and confirmed 1.16 runs OK on ARM64. This should be updated in another change to a supported version of Go at this time, I'll make another change to bump to 1.18.